src/modules/hyprland/workspaces.cpp

```diff
@@ -136,13 +167,25 @@ Workspaces::~Workspaces() {
   std::lock_guard<std::mutex> lg(mutex_);
 }
 
-WorkspaceDto WorkspaceDto::parse(const Json::Value &value) {
-  return WorkspaceDto{value["id"].asInt()};
-}
+Workspace::Workspace(const Json::Value &value)
+    : id_(value["id"].asInt()),
+      name_(value["name"].asString()),
+      output_(value["monitor"].asString()),  // TODO:allow using monitor desc
+      windows_(value["id"].asInt()) {
+  active_ = true;
+  is_special_ = false;
+
+  if (name_.find("name:") == 0) {
+    name_ = name_.substr(5);
+  } else if (name_.find("special") == 0) {
+    name_ = id_ == -99 ? name_ : name_.substr(13);
+    is_special_ = 1;
```

Comment:
`is_special = true`? )

Reply:
I changed this here: https://github.com/MightyPlaza/Waybar/pull/3